dftimewolf/lib/processors/turbinia_base.py

```diff
@@ -121,7 +121,7 @@ def _DownloadFilesFromGCS(self,
 
   def TurbiniaSetUp(self,
                     project: str,
-                    turbinia_recipe: str,
+                    turbinia_recipe: Union[str, None],
```

Comment:
No need to change but could you save the extra import and just do Optional[str].